src/components/calcite-stepper/calcite-stepper.stories.js

```diff
@@ -0,0 +1,162 @@
+import { storiesOf } from "@storybook/html";
+import { withKnobs, boolean, select, text } from "@storybook/addon-knobs";
+import { darkBackground, parseReadme } from "../../../.storybook/helpers";
+import readme from "./readme.md";
+const notes = parseReadme(readme);
+
+storiesOf("Stepper", module)
+  .addDecorator(withKnobs)
+  .add(
+    "Simple",
+    () => `
+    <calcite-stepper
```

Comment:
Does your editor auto-indent these to remove the actual tab indentation?

Reply:
I have had trouble getting VS Code to format these stories at all, with any formatter :(